javascript/UserForm.js

```diff
@@ -721,6 +721,9 @@ jQuery(function ($) {
 		setInterval(function () {
 			$.ajax({ url: 'UserDefinedForm_Controller/ping' });
 		}, 180 * 1000);
+
+		// Bind a confirmation message when navigating away from a partially completed form.
+		$('form.userform').areYouSure();
```

Comment:
Thanks for the quick response! This is indeed possible.

I will make that change soon and let you know.